components/layout_2020/flow/mod.rs

```diff
@@ -270,6 +350,9 @@ fn calculate_inline_content_size_for_block_level_boxes(
                 let style_box = &float_box.contents.style().get_box();
                 Some((size, style_box.float, style_box.clear))
             },
+            BlockLevelBox::OutsideMarker { .. } => {
+                Some((ContentSizes::zero(), Float::None, Clear::None))
+            },
```

Comment:
I think this could just use `None` together with the `OutOfFlowAbsolutelyPositionedBox`.
Since `::marker` appears at the beginning the behavior should be the same.

Reply:
Oh, good point. Adjusted this.